adlfs/spec.py

```diff
@@ -1474,6 +1491,10 @@ async def _url(self, path, expires=3600, **kwargs):
             permission=BlobSasPermissions(read=True),
             expiry=datetime.utcnow() + timedelta(seconds=expires),
             version_id=version_id,
+            content_disposition=content_disposition,
+            content_encoding=content_encoding,
+            content_language=content_language,
+            content_type=content_type,
```

Comment:
In s3fs this is supported by passing `**kwargs` directly into the aws/boto call, but it seems safer to be more explicit rather than passing kwargs into the `generate_blob_sas`.

Reply:
`gcsfs.sign()` also just passes `**kwargs` directly into the signing call, so I would not be opposed to making that change here if it's preferred